File: Problem1768_MergedStringAlternatively/1768.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int w1_s = word1.size();
        int w2_s = word2.size();
        int length = (w1_s>w2_s) ? w1_s : w2_s;
        string merged;
        
        for(int i = 0 ; i < length;++i){
            if(i < w1_s ){
                merged += word1[i];
                
            } if(i < w2_s){
                merged+=word2[i];
              
            }
        }
        return merged;

    }
};

File: LeetCode75/ArrayString/Problem1768_MergedStringAlternatively/1768.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        size = len(word1) if len(word1) > len(word2) else len(word2)
        ans = ''
        i = 0
        while(i < size):
            if(i < len(word1)):
                ans += word1[i]
            if(i < len(word2)):
                ans += word2[i]
            i = i+1
        return ans

